src/features/Travel/Main.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import Helmet from 'react-helmet';
+
+import { Hacker, Travel } from '../../api';
+import { HACKATHON_NAME, IHacker, ITravel } from '../../config';
+import { BackgroundImage, H1, H2, MaxWidthBox, Button } from '../../shared/Elements';
+
+import ValidationErrorGenerator from '../../shared/Form/validationErrorGenerator';
+import WithToasterContainer from '../../shared/HOC/withToaster';
+
+import Train from '../../assets/images/train.svg';
+
+export interface ITravelState {
+  hacker: IHacker | null;
+  travel: ITravel | null;
+  isLoading: boolean;
+}
+
+/**
+ * Container that renders form to log in.
+ */
+class TravelContainer extends React.Component<{}, ITravelState> {
+  constructor(props: {}) {
+    super(props);
+    this.state = {
+      hacker: null,
+      travel: null,
+      isLoading: true
+    };
+  }
+  public render() {
+    let reimbursement = <div></div>;
+    if (this.state.travel) {
+      switch (this.state.travel.status) {
+        case 'None':
+          reimbursement = <div>
+            Your request to recieve ${this.state.travel.request.toFixed(2)} in reimbursement for travel is still being processed.
+          </div>;
+          break;
+        case 'Bus':
+          reimbursement = <div>
+            You are taking a bus. We should put more info about the bus here.
```

Comment:
I would move this to case 'None' for now b/c we're opening the bus to everyone at first-come first-serve. So another section under status saying:

**Bus**
We're offering a bus route from Toronto to Montreal. Seats are offered on a first-come first-serve basis to all hackers. To secure a seat on the bus, please place a bus deposit here.